src/components/select/select.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+interface Props extends React.DetailedHTMLProps<React.SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement> {
+  options: Array<string>
+}
+
+const Select: React.FC<Props> = (props: Props) => {
+  const { options } = props
+
+  return (
+    <div className="inline-block relative w-64">
+      <select
+        className="block appearance-none w-full bg-zinc-900 text-white border-2 border-gray-400 hover:border-gray-500 px-4 py-2 pr-8 rounded shadow leading-tight focus:outline-none focus:shadow-outline focus:border-purple-600"
+        role={props.name}
+        {...props}
+      >
+        {options.map((option, i) => (
+          <option
+            key={i}
```

Comment:
É uma má prática utilizar keys de maneira generica assim. Por exemplo, a key nesse caso seria o numero daquela iteração, como 1,2,3. Se você tiver outro map na sua página, é possível que dois elementos diferentes tenham keys iguais. Tente sempre concatenar, a não ser que seja um id UUID
```suggestion
            key={`option-${key}`}
```

Reply:
Realmente, não me atentei por esse ponto, mas fiquei com uma dúvida... Nesse caso, por mais que eu utilize:
`option-${key}` 

Se eu utilizar vários selects na minha tela, ainda daria problema pois o prefixo "option-" seria igual em todos, não? Nesse caso se torna obrigatório a utilização de um gerador de UUID?

Reply:
commit: b1b7ed149602f930ee62c0d3f2e3349fd1fe848d

Reply:
Concatene de forma que não fiquem iguais, não use option, use o nome daquilo que representa por exemplo.